packages/fluentui/CHANGELOG.md

```diff
@@ -43,6 +43,7 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 - Removed `themes` export, each theme is exported separately @layershifter ([#13268](https://github.com/microsoft/fluentui/pull/13268))
 - Restricted prop sets in the `HeaderDescription` component which are passed to styles functions, @assuncaocharles ([#13296](https://github.com/microsoft/fluentui/pull/13296))
 - Restricted prop sets in the `Loader` component which are passed to styles functions, @assuncaocharles ([#13297](https://github.com/microsoft/fluentui/pull/13297))
+- Restricted prop sets in the `PortalInner` component which are passed to styles functions, @assuncaocharles ([#13298](https://github.com/microsoft/fluentui/pull/13298))
```

Comment:
```suggestion
```

Actually, here is no changes. `PortalInner` is a utility component without any styles